### PR TITLE
heavily touches up icebox old station

### PR DESCRIPTION
**Pull request by Addust**
h

## About The Pull Request

image
TL;DR: icebox oldstation has a TEG instead of solars now, icemoon turfs instead of airless, more consistent window breakages with what players can do, hopefully stops portals from spawning in, baseturf balloons deployed
WRECKAGE OF ALPHA STATION
image
Adds a tiny bit of alpha station with a few crates and rations, alongside a bear cave and a cargonian corpse.
CHARLIE STATION
image
there's a TEG now instead of solars, also gas miners in atmos too since they're cool
image
DELTA STATION
image
image
RnD server dismantled since prespawned RnD servers break for some bullshit reason and an airlock
EXTERIOR
image
image
some snowed plating in places

## Why It's Good For The Game

fixes some unwanted airless turfs on an ice ruin, TEG is cool, icebox oldstation was originally shit but now it's less shit
also makes the place easier to maintain

## Changelog
:cl:
add: Icebox Oldstation has been heavily touched up, including a TEG! have fun
fix: no more icebox oldstation airless turfs
fix: also gives it baseturfs
/:cl:

### CODE REVIEW

**Comment by Addust** [comment]
the old solar control one will be replaced with an APC controller (good luck unlocking it)
bridge will get some more consoles
xenobio will be added in a similar way to how it was on /tg/
